### DISCUSSION TITLE
honister branch builds 5.14.21 kernel from Poky

### DISCUSSION THREAD

**Discussion by jamse101** [superna9999/meta-meson#130] [Q&A]
I tried building meta-meson honister branch for hardkernel-odroidc4 and instead of 5.15 kernel 
I got 5.14.21.

I noticed that there was in meta-meson/conf/layer.conf:

BBFILE_PRIORITY_meson = "1"

But after changing that to 6 it still fetches 5.14.21.

**Comment by jamse101** [comment]
Using hardknott branch it fetches and builds 5.15.4.

**Comment by superna9999** [comment]
Hi it's expected since https://github.com/superna9999/meta-meson/commit/fbc88c1276a281633503c59db004cd8d30945e2a.

To use `5.15.4` you can add ` PREFERRED_PROVIDER_virtual/kernel = "linux-meson64"` in `local.conf`

**Reply by jamse101** [reply]
So, should I keep the BBFILE_PRIORITY as 1 or change it to 6 like it is in hardknott branch?

**Reply by superna9999** [reply]
BBFILE_PRIORITY is set as 1 in all branches, keeping it to 1 should be ok

**Reply by jamse101** [reply]
Ok, it seems that I've previously changed it in hardknott branch from 1 to 6 and then forgot about it :)
